solr/core/src/test/org/apache/solr/handler/api/V2ApiUtilsTest.java

```diff
@@ -36,4 +40,21 @@ public void testReadsDisableV2ApiSysprop() {
     assertFalse(
         "v2 API should be disabled if sysprop explicitly disables it", V2ApiUtils.isEnabled());
   }
+
+  @Test
+  public void testConvertsWtToMediaTypeString() {
+    assertEquals(
+        "someDefault",
+        V2ApiUtils.getMediaTypeFromWtParam(new ModifiableSolrParams(), "someDefault"));
+
+    var params = new ModifiableSolrParams();
+    params.add("wt", "json");
+    assertEquals(MediaType.APPLICATION_JSON, V2ApiUtils.getMediaTypeFromWtParam(params, null));
+
+    params.set("wt", "xml");
+    assertEquals(MediaType.APPLICATION_XML, V2ApiUtils.getMediaTypeFromWtParam(params, null));
+
+    params.set("wt", "javabin");
+    assertEquals(BINARY_CONTENT_TYPE_V2, V2ApiUtils.getMediaTypeFromWtParam(params, null));
+  }
```

Comment:
params.set(wt, foo)

Reply:
Can you clarify? Not sure what you're trying to convey here...

Reply:
You're missing test coverage for an unknown type